packages/cx/src/widgets/form/UploadButton.js

```diff
@@ -85,6 +85,7 @@ class UploadButtonComponent extends VDOM.Component {
                   className={CSS.element(baseClass, "input")}
                   type="file"
                   title=" "
+                  accept={widget.accept}
```

Comment:
Make the field bindable. Register `accept` in `declareData` and use `data.accept` here. 
I can imagine scenarios where this value should be calculated based on some input elements in the same form.